device/workflow.go

```diff
@@ -43,41 +45,47 @@ func New(device string) (Flasher, error) {
 			}
 		}
 
-		if len(g) == 0 {
-			return nil, errors.New("No custom boards are available")
+		if len(g) != 0 {
+			device = g[dialogs.SelectOneDialog("Please select your custom board: ", g)]
 		}
 
-		device = g[dialogs.SelectOneDialog("Please select a cutom board: ", g)]
-
 	}
+	r := &repo.DeviceMapping{}
```

Comment:
__[gometalinter]__
* [ineffassign] ineffectual assignment to r

<!-- Comment by [SideCI](https://sideci.com) -->